site/sass/components/_stories-section.scss

```diff
@@ -0,0 +1,14 @@
+// Styles for stories section component.
+@use 'sass:math';
+
+.stories-section {
+  $max-width: 1180px;
```

Comment:
Because this is disconnected from the featured stories styling, these two numbers are different and the final results don't align (see screenshot). Reusing the `max-width` we already have in the `wrapper` component would resolve this and make sure they're aligned; it's that component's raison d'etre 😝 .

This may mean that our content grid expands to 3 items when very large, and that's OK. I believe the `aspect-ratio` comment above will fix the visual issues when the cards are condensed.

![unaligned cards and featured story area](https://user-images.githubusercontent.com/377188/159529275-a25dc970-aa78-4f29-8258-484b12181127.png)

Reply:
raison d'etre lol

Removed.